Cinema-Reservation-System-Front-End-Client-Part-Internship-project/src/app/authentication/register/register.component.ts

```diff
@@ -0,0 +1,84 @@
+import { Component } from '@angular/core';
+import { ButtonComponent } from '../../shared/button/button.component';
+import { InputFieldComponent } from '../../shared/input-field/input-field.component';
+import {
+  FormBuilder,
+  FormGroup,
+  ReactiveFormsModule,
+  Validators,
+} from '@angular/forms';
+import { AuthenticationService } from '../shared/authentication.service';
+
+@Component({
+  selector: 'app-register',
+  standalone: true,
+  imports: [ButtonComponent, InputFieldComponent, ReactiveFormsModule],
+  templateUrl: './register.component.html',
+  styleUrl: './register.component.css',
+})
+export class RegisterComponent {
+  buttonLabel: string = 'Register';
+  placeHolders = {
+    firstName: 'First Name',
+    lastName: 'Last Name',
+    email: 'Email',
+    password: 'Password',
+    confirmPassword: 'Confirm Password',
```

Comment:
There is no sense to keep the placeholders here unless we want change them dynamically or change the controls of the form dynamically, lets move the placeholders to the template